modules/ossm-smcp-prod.adoc

```diff
@@ -17,14 +17,14 @@ You cannot change the `metadata.name` field of an existing `ServiceMeshControlPl
 .. Edit the `ServiceMeshControlPlane` resource to use the `production` deployment strategy, by setting `spec.addons.jaeger.install.storage.type` to `Elasticsearch` and specify additional configuration options under `install`. You can create and configure your Jaeger instance and set `spec.addons.jaeger.name` to the name of the Jaeger instance.
 +
 .Default Jaeger parameters including Elasticsearch
-[source,yaml]
+[source,yaml, subs="attributes,verbatim"]
 ----
 apiVersion: maistra.io/v2
 kind: ServiceMeshControlPlane
 metadata:
   name: basic
 spec:
-  version: v2.1
+  version: v/{MaistraVersion}
```

Comment:
is the "/" here intentional? I notice the other yaml files changed in this PR don't have it

Reply:
No, thanks for catching that.  I think that was a leftover from when I was still trying to figure out what the "escape" was to get variables to work in code examples.  Fixed!